### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by perguth**
Some changes here and there for consistency, clarity and readability. ✨

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -414,11 +414,11 @@ Emitted when a data block has been downloaded.
 
 #### `feed.on('upload', index, data)`
 
-Emitted when a data block is uploaded.
+Emitted when a data block has been uploaded.
```
Comment (perguth) [inline README.md]: I am guessing that though.

File: README.md [hunk 2]
```diff
@@ -134,7 +134,7 @@ Callback is called with `(err, data)`
 
 #### `feed.getBatch(start, end, [options], callback)`
 
-Get a range of blocks efficiently. Options include
+Get a range of blocks efficiently. End index is non-inclusive. Options include
```
Comment (emilbayes) [inline README.md]: Another word for "non-inclusive" could be "exclusive"

File: README.md [hunk 3]
```diff
@@ -352,7 +352,7 @@ Populated after `ready` has been emitted. Will be `null` before the event.
 
 #### `feed.discoveryKey`
 
-Buffer containing a key derived from the feed.key.
+Buffer containing a key derived from the feeds public key.
```
Comment (emilbayes) [inline README.md]: I think feeds should have an apostrophe on the end to be grammatically correct, as it is possessive. Ie. `from the feeds' public key`

File: README.md (hunk 1 above)
Comment (emilbayes) [reply README.md]: I actually think the former is correct as it happens *before* the block is sent over the wire. See https://github.com/mafintosh/hypercore/blob/5a46735bb95e96265dea32f2919a94da120b00fa/lib/replicate.js#L181-L195

**Comment by emilbayes** [comment]
Looks good! Added a few comments :)

File: README.md (hunk 2 above)
Comment (perguth) [reply README.md]: It's strange but somehow I feel a novice would understand `non-inclusive` more easily. But As you say.

Comment (emilbayes) [reply README.md]: @perguth I don't have any power here, in math/CS I feel like intervals are usually called inclusive and exclusive

File: README.md (hunk 1 above)
Comment (perguth) [reply README.md]: Ok, clarified. Ty!

File: README.md (hunk 2 above)
Comment (perguth) [reply README.md]: Ok, I would stick to non-inclusive. It's also used a couple of lines down in this paragraph: https://github.com/mafintosh/hypercore/blob/5a46735bb95e96265dea32f2919a94da120b00fa/README.md#feeddownloadrange-callback

**Comment by perguth** [comment]
Good for merge then. Thanks everyone! 🚀

**Comment by perguth** [comment]
Ping. 🏓

**Comment by perguth** [comment]
thanks 😊

**Pull request merged**